SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -168,6 +168,11 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	UPROPERTY(config)
 	bool bEnableServerQBI;
 
+	/** EXPERIMENTAL - Adds granular result types for client queries.
```

Comment:
Describe what granular means here in more detail if you can, specifically that you are including all components and the GDK's WKC

Reply:
Added description of what granular means